cmd/operator-sdk/main.go

```diff
@@ -51,6 +51,16 @@ func main() {
 				"https://sdk.operatorframework.io/docs/golang/project_migration_guide/\n"
 			projutil.PrintDeprecationWarning(depMsg)
 		}
+
+		if operatorType == projutil.OperatorTypeHelm {
+			depMsg := "Operator SDK has a new CLI and project layout that is aligned with Kubebuilder.\n" +
+				"See `operator-sdk init --plugins=helm.operator-sdk.io/v1 -h` and the following doc on how to scaffold a new project:\n" +
+				"https://sdk.operatorframework.io/docs/helm/quickstart/\n" +
+				"To migrate existing projects to the new layout see:\n" +
+				"https://sdk.operatorframework.io/docs/helm/migration\n"
+			projutil.PrintDeprecationWarning(depMsg)
+		}
```

Comment:
IMO it would be helpful for who still using the old project and requires to migrate. However, we also will have this info in the migration guide and etc .. so IMO we could remove this msgs as well. 
So, +1 to remove that. WDYT?

Reply:
Yeah I agree.